README.md

```diff
@@ -45,6 +46,8 @@ predictions = model.predict(texts)
 # Verify Articles Classified Correctly 
 assert predictions == [1,0]
 ```
+ktrain may throw a UserWarning which you can safely ignore. 
```

Comment:
Would make this a comment in the code snippet, ideally right above the line that causes the warning